README.md

```diff
@@ -2,29 +2,36 @@
 # parcelLab SRE challenge - “Hello World”
 
 - Create a “Hello World” greetings service (REST-API) which responds with different salutations for customers (there won’t be further purpose for this API, just one GET endpoint is fine)
-    - Customer A wants to say “Hi”, customer B wants to say “Dear Sir or Madam”, customer C wants to say “Moin” and so on.
-    - Choose whatever programming language and framework you think might be the best fit
-    - Describe or implement the necessary things to ensure “good quality“ for your product
-- Create a build pipeline/script to create a docker image for this API and register it at a container registry (local registry is fine)
+    - Customer A wants to say “Hi”, customer B wants to say “Dear Sir or Madam”, customer C wants to say “Moin” and so on. The system should identify users based on the salutation they send to the endpoint.
```

Comment:
This is actually the point - the code should be the same for all instances. Only the configuration for each customer's instance should be adjusted. Maybe "The system should identify the users based on the url the are requesting" ?